### DISCUSSION TITLE
[Feat]: Autoplay - Slide to reverse direction

### DISCUSSION THREAD

**Discussion by wrspada02** [davidjerleke/embla-carousel#821] [Ideas]
### Which variants of Embla Carousel are relevant to this feature request?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Please It'd be nice adding the feature to slide on the reverse direction.

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by sarussss** [comment]
Hi @wrspada02
I think it might be useful to you.
 [direction]( https://www.embla-carousel.com/api/options/#direction)
 
 **Updated:** My mistake for not paying attention to the question about the autoplay plugin.
 **Please don't mind this comment.**

**Reply by wrspada02** [reply]
Thanks for replying @sarussss !!

**Comment by davidjerleke** [comment]
@wrspada02 this is easy to add but if [this option](https://www.embla-carousel.com/plugins/autoplay/#stoponlastsnap) is set to `true` and it’s a carousel with loop disabled, you realize that autoplay won’t do anything?

**Reply by sarussss** [reply]
Hi @davidjerleke.
Can we take advantage of [direction option](https://www.embla-carousel.com/api/options/#direction) for this case?
[Autoplay Reverse Demo](https://stackblitz.com/edit/sarusss-autoplay-d4adcr?file=main.js)

**Reply by davidjerleke** [reply]
@sarussss thanks for helping out! I don't think so because the [direction option](https://www.embla-carousel.com/api/options/#direction) is there for devs who use `RTL` (right to left) languages on their website. However, when `direction` is set to `rtl`, Autoplay correctly adapts to that and slides from right to left.

I think @wrspada02 wants Autoplay to play backwards, meaning `scrollPrev` instead of `scrollNext`.

**Reply by wrspada02** [reply]
First of all, thanks for replying to my request @sarussss @davidjerleke.

I saw the @sarussss demo looks good in the reverse way. I'll try it out.

@davidjerleke I tried yesterday to add the rtl mode to my carousel. But, when it swipes, the carousel just displays 3 cards and the remaining cards stop showing up. But, I didn't apply this note on my components there's on the documentation.

![image](https://github.com/davidjerleke/embla-carousel/assets/90157791/3427f262-3964-4895-8ad3-b57f4f16f4ef)

The feature I requested itself would be to the plugin. Just pass an option to the plugin and it right away starts to run in reverse mode.

I could help in developing that.

**Reply by davidjerleke** [reply]
@wrspada02 I've added your idea to the roadmap:

- https://github.com/davidjerleke/embla-carousel/discussions/841
